### PR TITLE
Renovation: implement the Droppable component

### PR DESCRIPTION
**Pull request by Alyar666**

### CODE REVIEW

File: js/renovation/ui/droppable.tsx [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+  @Event() onDragLeave?: (e: DroppableEvent) => void;
+
+  @Event() onDrop?: (e: DroppableEvent) => void;
+}
+
+@Component({
+  defaultOptionRules: null,
+  view: viewFunction,
+  jQuery: {
+    register: true,
```
Comment (vconst) [inline js/renovation/ui/droppable.tsx]: Remove jquery registration if it is internal widget now.

File: js/renovation/ui/droppable.tsx [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+  itemElement: HTMLDivElement;
+}
+
+@ComponentBindings()
+export class DroppableProps {
+  @OneWay() disabled?: boolean;
+
+  @OneWay() className = '';
+
+  @Slot() children?: JSX.Element | (JSX.Element | undefined | false | null)[];
```
Comment (MikeVitik) [inline js/renovation/ui/droppable.tsx]: I suppose it simple JSX.Element without (false | null)[]

File: js/renovation/ui/droppable.tsx [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+);
+
+interface DroppableEvent {
+  event: Event;
+  itemElement: HTMLDivElement;
+}
+
+@ComponentBindings()
+export class DroppableProps {
+  @OneWay() disabled?: boolean;
```
Comment (MikeVitik) [inline js/renovation/ui/droppable.tsx]: Do you miss default - false?

File: js/renovation/ui/droppable.tsx [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+
+  @OneWay() className = '';
+
+  @Slot() children?: JSX.Element | (JSX.Element | undefined | false | null)[];
+
+  @Event() onDragEnter?: (e: DroppableEvent) => void;
```
Comment (MikeVitik) [inline js/renovation/ui/droppable.tsx]: Replace to @Event() onDragEnter?: EventCallback<DroppableEvent>;

**Review by MikeVitik — CHANGES_REQUESTED** [review]

**Review by MikeVitik — APPROVED** [review]

**Pull request merged**